docs/concepts/abstractions/controllers/garbage-collection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Garbage Collection
+---
+
+{% capture overview %}
+
+When you delete a Kubernetes object, the garbage collector deletes all
+of the objects' dependents. For example suppose you delete a ReplicaSet
+that owns several Pods. When you delete the ReplicaSet, the garbage collector
+deletes all Pods owned by the ReplicaSet.
+
+**Note**: Garbage collection is a beta feature and is enabled by default in
+Kubernetes version 1.4 and later.
+
+{% endcapture %}
+
+
+{% capture body %}
+
+## Owners and dependents
+
+Some Kubernetes objects are owners of other objects. For example, a ReplicaSet
+is the owner of a set of Pods. The owned objects are called *dependents* of the
+owner object. Every dependent object has a `metadata.ownerReferences` field that
+points to the owning API object.
+
+Sometimes, Kubernetes sets the value of `ownerReference` automatically. For
+example, when you create a ReplicaSet, Kubernetes automatically sets the
+`ownerReference` field of each Pod created for the ReplicaSet. You can also
+specify relationships between owners and dependents by manually setting the
+`ownerReference` field.
+
+## Controlling whether the garbage collector deletes dependents
```

Comment:
Sorry, its not the title is misleading, it's the description. OrphanDependents means "no garbage collection".